aiida/work/legacy/job_process.py

```diff
@@ -151,3 +151,9 @@ def _setup_db_record(self):
 
         if parent_calc:
             self._calc.add_link_from(parent_calc, "CALL", LinkType.CALL)
+
+        if self.raw_inputs:
```

Comment:
Actually I think I've seen the problem happen with ``Process``, but at the time just assumed it wasn't supposed to work with ``get_inputs_template``.